hail/src/main/scala/is/hail/expr/ir/BlockMatrixIR.scala

```diff
@@ -399,9 +429,10 @@ case class BlockMatrixBroadcast(
       case IndexedSeq(1) =>
         BlockMatrixIR.checkFitsIntoArray(nRows, nCols)
         broadcastRowVector(hc, childBm.toBreezeMatrix().data, nRows.toInt, nCols.toInt)
-      case IndexedSeq(0, 0) =>
-        BlockMatrixIR.checkFitsIntoArray(nRows, nCols)
-        BlockMatrixIR.toBlockMatrix(hc, nRows.toInt, nCols.toInt, childBm.diagonal(), blockSize)
+        // FIXME: I'm pretty sure this case is broken.
```

Comment:
need a 0,0 case?

Reply:
I'm pretty sure that the (0, 0) case is both broken (incorrectly implemented) and unused, so I chose to comment it out entirely to indicate that it's not supported. (I don't think the lack of match on that is going to be problematic unless there's a test somewhere that I didn't see; I'm pretty sure this node is also missing the (1, 1) case which would also be technically valid.)